### ISSUE TITLE
C/S Error

### ISSUE THREAD

**Issue by hdoupe** [PSLmodels/OG-USA#578]
Inputs:

```json
{
    "meta_parameters": {
        "year": 2020,
        "data_source": "CPS",
        "time_path": true
    },
    "adjustment": {
        "Tax-Calculator Parameters": {
            "EITC_MinEligAge": [
                {
                    "value": 18,
                    "year": 2020
                }
            ],
            "EITC_ps_MarriedJ": [
                {
                    "value": 9000,
                    "EIC": "3+kids",
                    "year": 2020
                },
                {
                    "value": 9000,
                    "EIC": "2kids",
                    "year": 2020
                },
                {
                    "value": 9000,
                    "EIC": "1kid",
                    "year": 2020
                },
                {
                    "value": 9000,
                    "EIC": "0kids",
                    "year": 2020
                }
            ],
            "EITC_ps": [
                {
                    "value": 24000,
                    "EIC": "3+kids",
                    "year": 2020
                },
                {
                    "value": 21000,
                    "EIC": "2kids",
                    "year": 2020
                },
                {
                    "value": 18000,
                    "EIC": "1kid",
                    "year": 2020
                },
                {
                    "value": 15000,
                    "EIC": "0kids",
                    "year": 2020
                }
            ],
            "EITC_prt": [
                {
                    "value": 0.25,
                    "EIC": "3+kids",
                    "year": 2020
                },
                {
                    "value": 0.25,
                    "EIC": "2kids",
                    "year": 2020
                },
                {
                    "value": 0.25,
                    "EIC": "1kid",
                    "year": 2020
                },
                {
                    "value": 0.25,
                    "EIC": "0kids",
                    "year": 2020
                }
            ],
            "EITC_basic_frac": [
                {
                    "value": 0.67,
                    "year": 2020
                }
            ],
            "EITC_rt": [
                {
                    "value": 0.25,
                    "EIC": "3+kids",
                    "year": 2020
                },
                {
                    "value": 0.25,
                    "EIC": "2kids",
                    "year": 2020
                },
                {
                    "value": 0.25,
                    "EIC": "1kid",
                    "year": 2020
                },
                {
                    "value": 0.25,
                    "EIC": "0kids",
                    "year": 2020
                }
            ],
            "EITC_c": [
                {
                    "value": 18000,
                    "EIC": "3+kids",
                    "year": 2020
                },
                {
                    "value": 15000,
                    "EIC": "2kids",
                    "year": 2020
                },
                {
                    "value": 12000,
                    "EIC": "1kid",
                    "year": 2020
                },
                {
                    "value": 9000,
                    "EIC": "0kids",
                    "year": 2020
                }
            ]
        },
        "OG-USA Parameters": {
            "start_year": [
                {
                    "value": 2022
                }
            ],
            "tG1": [
                {
                    "value": 80
                }
            ]
        }
    }
}
```

- Traceback:

```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/cs_workers/models/executors/task_wrapper.py", line 69, in async_task_wrapper
    outputs = func(task_id, **(task_kwargs or {}))
  File "/opt/conda/lib/python3.7/site-packages/cs_workers/models/executors/job.py", line 17, in sim_handler
    outputs = functions.run_model(meta_param_dict, adjustment)
  File "/opt/conda/lib/python3.7/site-packages/cs_config/functions.py", line 233, in run_model
    reform_tpi = TPI.run_TPI(reform_params, client=client)
  File "/home/OG-USA/ogusa/TPI.py", line 740, in run_TPI
    raise RuntimeError('Transition path equlibrium not found ' +
RuntimeError: Transition path equlibrium not found (RC_error)
```

**Comment by jdebacker** [comment]
@hdoupe Thanks for raising this issue.  It looks like perhaps the EITC here is too generous and that is causing some issues with the budget along the transition path.  But I'll look into this in more detail and report back.  

Perhaps I need to think about how we can report a more detailed message about what caused the failure.

cc @rickecon

**Comment by jdebacker** [comment]
@hdoupe Ok - running this locally and looking at output, it is the case that this policy is too generous and causes significant budget issues.  The debt-to-GDP ratio exceeds 5000% before the closure rule kicks in 2102.  At that point, the government is consuming more resources than exist in the economy, which leads a violation of the resource constraint (`RC_error`) noted in the traceback.

In doing this, I did notice that the inputs have conflicting years:
```
    "meta_parameters": {
        "year": 2020,
        "data_source": "CPS",
        "time_path": true
    },
```
and

```
"OG-USA Parameters": {
            "start_year": [
                {
                    "value": 2022
                }
            ],
```

I'll open a PR so the user can only enter a year in one place.

**Comment by hdoupe** [comment]
Thanks for the info @jdebacker. I'll pass this on to the user.

Your fix for only specifying the start year in one place makes sense to me.

**Comment by jdebacker** [comment]
This issue was resolved with PR #599.
